src/dive/ui/components/settings/settings.cpp

```diff
@@ -89,7 +91,7 @@ Settings::DisplayUnit Settings::ReadRulerDisplayUnit()
 {
     QSettings   settings;
     DisplayUnit display_unit = (DisplayUnit)settings.value("rulerDisplayUnit", 0).toInt();
-    DIVE_ASSERT(display_unit == kCycle || display_unit == kMs || display_unit == kUs ||
+    ABSL_ASSERT(display_unit == kCycle || display_unit == kMs || display_unit == kUs ||
```

Comment:
would it be better to keep DIVE_ASSERT and change it to call ABSL_ASSERT internally? otherwise you need to replace all DIVE_ASSERTs in Dive proj which could be quite a lot